core/templates/dev/head/services/CodeNormalizerSpec.js

```diff
@@ -0,0 +1,123 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
```

Comment:
This file isn't named correctly. It should be CodeNormalizerServiceSpec.

Reply:
Done.